test/parallel/test-event-emitter-remove-all-listeners.js

```diff
@@ -24,7 +24,6 @@ const common = require('../common');
 const assert = require('assert');
 const events = require('events');
 
-
```

Comment:
Whoops, unintentional style change. I think I had written something there and then deleted too much.